2206.divide-array-into-equal-pairs.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/divide-array-into-equal-pairs/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {boolean}
+ */
+var divideArray = function(nums) {
+    let counter = 0;
+    
+    const sortNumbers = nums.sort()
+    
+    for (let i = 1; i < sortNumbers.length; i = i + 2) {
```

Comment:
```suggestion
    const sortedNumbers = nums.sort()
    
    for (let i = 1; i < sortedNumbers.length; i = i + 2) {
```
I think this is more self-descriptive